.vscode/tasks.json

```diff
@@ -36,7 +41,10 @@
             "label": "npm: build:compiler",
             "type": "npm",
             "script": "build:compiler",
-            "group": "build",
+            "group": {
+                "kind": "build",
+                "isDefault": true

```

Comment:
Can you undo this unrelated change?

Reply:
Right this was never intended to be merged. Reverted.